WTraining/app/services/weet.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const axios = require('axios');
```

Comment:
We can to create a request helper to do requests external apis in order to wrap the axios package to be used to any service.